.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,44 @@
+name: Continuous Benchmarks with Bencher
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+
+jobs:
+  benchmark_with_bencher:
+    name: Benchmark with Bencher
+    permissions:
+      pull-requests: write
+    runs-on: ubuntu-latest
+    env:
+      BENCHER_PROJECT: core-crypto-mmbtki3h
+      BENCHER_API_TOKEN: ${{ secrets.BENCHER_API_TOKEN }}
+      BENCH_PARAMS: "--bench commit --bench encryption --bench key_package --bench create_group -- --quick"  # Run everything except mls_proteus
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions-rust-lang/setup-rust-toolchain@v1  # this implicitly caches Rust tools and build artifacts
+      - name: Install bencher
+        uses: bencherdev/bencher@main
+      - name: Run benchmarks on PR
+        if: github.event.pull_request.head.repo.full_name == github.repository && github.event_name == 'pull_request'
```

Comment:
Why is this line needed?

Reply:
Ah, I think I see why -- for PRs, we want to use the previously-measured `develop` as the baseline, right?
And then the second case below is just for pushes to `develop`, to create that baseline measurement?

Reply:
Yep, correct!